### ISSUE TITLE
make sure that the EMO BON url and the Handbook appears in the ENA entries

### ISSUE THREAD

**Issue by isanti** [emo-bon/governance-data#10]

**Comment by kmexter** [comment]
Ioulia - in which fields should these be? (I agree they should be there!) and which URL (given that the emobon page on the embrc site has minimal content)?
On a related issue, we discussed that the SOPs (samp_mat_process) given in the logsheets should be turned into URLs in the ttl files. However, as the SOPs are all pushed together in one Handbook, we cannot "turn the SOPs into URLs". What we will do in the ttl files then is for each SOP value (e.g. SoSOP3) we will add a property "can be found in this DOI". 
HOWEVER, that means I need to ask you this: when the SOPs are updated, will the Handbook update get a new DOI or will it use the old DOI? It is important that when we say "SoSOP3 can be found in [this doi]" it always refers to the handbook relevant at the time of sampling, and so if the Handbook is updated - or rather, if the SOPs in the handbook are updated - then they get a new DOI.

**Comment by cpavloud** [comment]
In https://github.com/emo-bon/sequencing-data/blob/main/shipment/batch-001/run-information-batch-001_column-descriptions.csv 
there is the **url_seq** and the **sop_seq**

When the Handbook is updated, it can be updated also in OBPS (it will appear as version 2) and get a new DOI.
We didn't do it for the update we did in April, but it can be done. 

However, we did update the Handbook, but basically changed only the ARMS part, so the other SOP have remained the same.

**Comment by cpavloud** [comment]
**url_seq = NA** is in the logsheets?
I don't know, maybe it was filled in before the Handbook had a DOI?

**Comment by kmexter** [comment]
I think this was for the url to the sops for the sequences, not for the collection. Is the sequence processing protocols also in the handbook? because the column sop_seq is just a string, not a URL -> so useless from a data POV

**Comment by cpavloud** [comment]
Yes, the sequencing protocols are included in the updated (2024) Handbook.

**Comment by kmexter** [comment]
FYI the handbook on OBPS is version 1 from April 2021......see https://repository.oceanbestpractices.org/handle/11329/1738
Is there a version 2 or did someone just forget to update that information on the front page of the handbook PDF?

**Comment by kmexter** [comment]
OK, well in order to be able to reference a DOI rather than a URL, we will point to the 2021 version (well, always to the version in OBPS). Can you tell me when you do update OBPS please?

**Comment by cpavloud** [comment]
Of course!

**Comment by cpavloud** [comment]
This is relevant to https://github.com/emo-bon/sequencing-data/issues/16 and https://github.com/emo-bon/governance-data/issues/1
